src/dom/dom.js

```diff
@@ -2421,6 +2421,93 @@ p5.Element.prototype.drop = function (callback, fxn) {
   return this;
 };
 
+/**
+ * Turns p5.Element into a draggable item with the mouse. If argument is given, it will drag a different p5.Element (parent container) instead, ie. for a GUI title bar.
```

Comment:
Maybe we can describe the example's behaviour in a bit more detail here, something like: this lets you drag the title bar of a panel to move the whole panel around

Reply:
Noted and updated